src/ngx_http_graphite_module.c

```diff
@@ -1813,8 +1929,14 @@ ngx_http_graphite_timer_event_handler(ngx_event_t *ev) {
 
             if (nl > part) {
 
-                if (sendto(fd, part, nl - part + 1, 0, (struct sockaddr*)&sin, sizeof(sin)) == -1)
-                    ngx_log_error(NGX_LOG_ALERT, ev->log, 0, "graphite can't send udp packet");
```

Comment:
`if (result == EWOULDBLOCK) ` Doesn't make much sense. You should check if the `errno` is set to EWOULDBLOCK. When the result is negative, it means errno is set. If errno is set to EWOULDBLOCK, retry to send in the next iteration, file descriptor not ready for write.